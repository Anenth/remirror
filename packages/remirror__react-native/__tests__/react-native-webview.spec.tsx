import { rollupBundle, transpileFile } from 'bundler.macro';

test('it can transpile a file', () => {
  expect(transpileFile('../__fixtures__/transpile.fixture.ts')).toMatchSnapshot();
});

test('it can bundle a file', () => {
  expect(rollupBundle('../__fixtures__/bundle.fixture.ts')).toMatchSnapshot();
});

// test('it can bundle an editor', () => {
//   expect(rollupBundle('../__fixtures__/full.fixture.tsx')).toMatchSnapshot();
// });
