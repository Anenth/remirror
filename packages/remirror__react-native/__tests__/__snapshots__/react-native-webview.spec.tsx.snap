// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it can bundle a file 1`] = `
"var __ROLLUP_BUNDLER_MACRO__ = (function (exports) {
  'use strict';

  function fn() {
    return 'awesome';
  }

  const a = /*#__PURE__*/fn();

  exports.a = a;
  exports.default = fn;

  Object.defineProperty(exports, '__esModule', { value: true });

  return exports;

}({}));
"
`;

exports[`it can transpile a file 1`] = `
"function TestFunction() {
  console.log('This is a function');
}"
`;
